package.json

```diff
@@ -142,67 +143,46 @@
     "ts-node": "^8.10.2",
     "typescript": "^4.1.3"
   },
   "scripts": {
-    "build": "yarn build-standalone",
-    "build-mac": "yarn build-mac-from-archive",
-    "build-mac-from-archive": "scripts/build/installers/darwin_from_archive",
-    "build-mac-from-sources": "scripts/build/installers/darwin_from_sources",
-    "build-standalone": "scripts/build/standalone",
-    "build-windows": "scripts/build/installers/windows",
-    "pack:tarballs": "oclif pack:tarballs",
-    "pack:macos": "oclif pack:macos",
-    "pack:deb": "oclif pack:deb",
-    "pack:win": "oclif pack:win",
-    "upload:tarballs": "oclif upload:tarballs",
-    "upload:macos": "oclif upload:macos",
-    "upload:deb": "oclif upload:deb",
-    "upload:win": "oclif upload:win",
-    "promote": "oclif promote",
-    "clean": "yarn clean-dist && yarn clean-test",
-    "clean-all": "scripts/clean/all",
-    "clean-build": "scripts/clean/build",
-    "clean-compile": "yarn clean-all && yarn install --pure-lockfile",
-    "clean-dist": "scripts/clean/dist",
-    "clean-test": "scripts/clean/test",
-    "compile": "scripts/build/compile",
-    "coverage-report": "scripts/test/coverage",
+    "XXpostpack:tarballs": "./scripts/verify-tarballs",
+    "XXprepack:tarballs": "./scripts/clean-for-tarballs",
+    "build": "sf-build",
+    "clean": "shx rm -rf dist tmp",
+    "compile": "sf-compile",
     "docs": "sf-docs",
     "format": "sf-format",
-    "lint": "scripts/lint/sources",
-    "lint-all": "scripts/lint/all",
-    "lint-with-style": "scripts/lint/sources_with_style",
-    "oclif-artifacts": "oclif-dev manifest",
+    "lint": "eslint \"src/**/*.ts\"",
+    "oclif-artifacts": "oclif manifest",
+    "pack:deb": "oclif pack:deb",
+    "pack:macos": "oclif pack:macos",
+    "pack:tarballs": "oclif pack:tarballs",
+    "pack:win": "oclif pack:win",
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "yarn test:deprecation-policy ",
     "prepack": "yarn clean && yarn compile && yarn lint && yarn oclif-artifacts",
     "prepare": "yarn compile && yarn lint",
-    "release": "yarn release-all",
-    "release-all": "scripts/release/all",
-    "release-all-debug": "env SKIP_AWS=true yarn release-all",
-    "release-debug": "yarn release-all-debug",
-    "release-mac": "scripts/release/installers/darwin",
-    "release-mac-debug": "env SKIP_AWS=true yarn release-mac",
-    "release-standalone": "scripts/release/standalone",
-    "release-standalone-debug": "env SKIP_AWS=true yarn release-standalone",
-    "release-windows": "scripts/release/installers/windows_docker",
-    "release-windows-debug": "env SKIP_AWS=true yarn release-windows",
+    "promote": "oclif promote",
     "snapshot-generate": "HOME=./ ./bin/run snapshot:generate",
-    "test": "scripts/test/all",
-    "test:deprecation-policy": "HOME=./ ./bin/run snapshot:compare",
+    "test": "nyc --report lcov --report cobertura mocha -t 240000 --require source-map-support/register --recursive \"dist/**/*.test.js\"",
```

Comment:
what's gonna be in test/uts ?

Reply:
where do the cobertura stuff go?

Reply:
That is probably legacy stuff too. I wasn't sure so I just left it for now.